lms/djangoapps/ccx/views.py

```diff
@@ -398,8 +400,15 @@ def ccx_student_management(request, course):
             unenroll_email(ccx, email, email_students=False)
     except ValidationError:
         log.info('Invalid user name or email when trying to enroll student: %s', email)
+        if email:
+            error_message = 'Could not find a user with name or email "%s"'\
+                            % email
 
     url = reverse('ccx_coach_dashboard', kwargs={'course_id': course.id})
+    if error_message:
+        params = {'error_message': error_message}
+        return HttpResponseRedirect(url + "?%s" % urllib.urlencode(params))
```

Comment:
Rather than injecting data into another view, using the Django Messaging to transfer session messages seems like a good way to do this.

Here's an example of messaging in use within the platform:
View: https://github.com/jazkarta/edx-platform/blob/master/lms/djangoapps/dashboard/support.py#L122-L124
Template: https://github.com/jazkarta/edx-platform/blob/master/lms/templates/dashboard/_dashboard_refund.html#L28-L34